contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -322,6 +335,22 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
         );
     }
 
+    /**
+     * @notice Stakes a users vault shares
+     * @param numShares is the number of shares to stake
+     */
+    function stake(uint256 numShares) external nonReentrant {
+        require(stakingRewards != address(0)); // Removed revert msgs due to contract size limit
+        require(numShares > 0);
+        (uint256 heldByAccount, ) = shareBalances(msg.sender);
```

Comment:
If we're only interested in `heldByAccount`, this is equivalent but with less gas use:

```
uint256 heldByAccount = balanceOf(msg.sender);
```

Reply:
Oh nice, just changed it now